### ISSUE TITLE
White paper organization

### ISSUE THREAD

**Issue by phinate** [smoothep/center#6]
We agreed at the kick off meeting to form a white paper that highlights this fairly new analysis paradigm for people new to it.

I think this may end up becoming two papers: the ‘whys’ (initial motivations, existing efforts) and the ‘hows’ (evaluation and comparisons of the methods implemented with common tools in a realistic setting).

What should we include (in paper 1?) :)

**Comment by tdorigo** [comment]
I suppose we might have: 
- an intro with a discussion of the issue of nuisance parameters in physics measurements,  and methods proposed in the literature to limit their impact; here we could point to relevant / important papers and seminal studies
- a few sections, in each of which we discuss a different area of research, the goals that differentiable analysis tools may help achieve there, and the proposed tools under development that may make a difference there. E.g.:
- optimal classification (with inferno and neos approaches briefly discussed)
- optimal regression (what do we have here?)
- pareto optimization of detectors (this is a favourite of mine, and a grand plan... of course one needs to discuss first and foremost how to encode the desiderata of an experimental apparatus into a complete cost function)

**Comment by phinate** [comment]
Thanks for the list @tdorigo! A comment:

> * optimal classification (with inferno and neos approaches briefly discussed)
> * optimal regression (what do we have here?)

Not sure if these are accurate subcategories -- neither of neos or inferno are classifiers right? That to me implies training with a classification-based objective, which each approach explicitly tries to circumvent. A more accurate picture in my mind would be one of optimal summary statistics, or even in some sense optimal event selection. Optimising a differentiable cutflow could also come into play there.

**Comment by tdorigo** [comment]
I agree, but I would make the list of topics and proposed solutions correspond as closely as possible to the HEP use cases of interest, rather than display a list of tools.

**Comment by phinate** [comment]
Just for the sake of keeping track of the timescale, me and @lukasheinrich will try to publish something on neos in the next month or so before working on this, which will also allow us to cite it :)

**Comment by tdorigo** [comment]
Good, always cite yourself. If you don't, why expect others to do the same?

**Comment by GilesStrong** [comment]
To help lay the groundwork for part 2, it might be useful to layout in part 1 a set of ideal specifications for the hows (e.g. modular, intuitive & accessible, and low hardware-requirements). Whilst we might have the specification in mind ourselves, this could be useful for getting early feedback from the community prior to writing part 2, in case we miss anything, or certain things are redundant.
